Documentation/debugging/unix-instructions.md

```diff
@@ -6,10 +6,40 @@ CoreFX can be debugged on unix using both lldb and visual studio code
 ## Using lldb and SOS
 
 - Run the test using msbuild at least once with `/t:BuildAndTest`.
-- Install version 3.6 of lldb and launch lldb with corerun as the process and arguments matching the arguments used when running the test through msbuild.
+- Install version 3.6 of lldb and launch lldb with dotnet as the process and arguments matching the arguments used when running the test through msbuild.
 - Load the sos plugin using `plugin load libsosplugin.so`.
 - Type `soshelp` to get help. You can now use all sos commands like `bpmd`.
 
+## Debugging core dumps with lldb
+
+It is also possible to debug .NET Core crash dumps using lldb and SOS. In order to do this, you need all of the following:
+
+- A machine whose environment matches the one used to produce the crash dumps. For crash dumps occurring on CI machines, you can either
+  - Log onto a matching machine in the pool.
+  - Create a new VM from the matching image.
+- The crash dump file. We have a service called "Dumpling" which collects, uploads, and archives crash dump files during all of our CI jobs and official builds.
+- Matching runtime bits from the crash. To get these, you should either:
+  - Download the matching Jenkins archive onto your repro machine.
+  - Check out the corefx repository at the appropriate commit and re-build the necessary portions.
+- Symbols for libcoreclr.so. libcoreclr.so.dbg should be copied to your "runtime" folder. To get this file, you can:
+  - Download the matching "symbols" nuget package from myget.org. You want the same package version that is used to build corefx. There is a "Download Symbols" button in the myget UI for this purpose.
+  - Build coreclr at the matching commit. In order to determine which commit was used to build a version of libcoreclr.so, run the following command:
+    `strings libcoreclr.so | grep "@(#)"`
+
+Once you have everything listed above, you are ready to start debugging. You need to specify an extra parameter to lldb in order for it to correctly resolve the symbols for libcoreclr.so. Use a command like this:
+
+```
+lldb -O "settings set target.exec-search-paths <runtime-path>" --core <core-file-path> <dotnet-path>
+```
+
+- `<runtime-path>`: The path containing libcoreclr.so.dbg, as well as the rest of the runtime and framework assemblies.
+- `<core-file-path>`: The path to the core dump you are attempting to debug.
+- `<dotnet-path>`: The path to the dotnet executable, potentially in the `<runtime-path>` folder.
+
+lldb should start debugging successfully at this point. You should see stacktraces with resolved symbols for libcoreclr.so. At this point, you can run `plugin load <libsosplugin.so-path>`, and begin using SOS commands, as above.
```

Comment:
We should add here that the lldb plugin must match the lldb version. The plugin that's in the dotnet core packages was compiled with lldb-3.6, so it works only with that one. Otherwise, the correct lldb plugin needs to be built from coreclr sources against the lldb libraries of the specific version. 
But I believe that we should provide lldb plugin for all available versions of lldb and stop packaging just the version for lldb 3.6 in the dotnet core packages.